app/assets/stylesheets/custom.css.scss

```diff
@@ -226,3 +226,39 @@ h1 {
 .collection {
   padding: 3px;
 }
+
+.submit {
+  align: center;
+  float: left;
+  margin-left: 43%;
+  width: 75%
+}
+
+.form {
+  align: center;
+  float: left;
+  margin-left: 43%;
+  width: 75%
+}
+
+.order-dashboard-page {
+  align: center;
+  float: left;
+  margin-left: 23%;
+  width: 75%
+}
+
+.admin-links p {
+  border: 2px solid black;
+  border-radius: 5px;
+  align: center;
+  margin-left: 25%;
+  margin-right: 25%;
+  text-align: center;
+  padding: 2px;
+}
+
+.admin-links p:hover {
+  background-color: grey;
+  font-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Unknown property font-color